src/resources/application.properties

```diff
@@ -9,3 +9,7 @@ logging.level.root=info
 logging.level.org.springframework.web=info
 logging.file.path=.
 logging.type=file
+#Ignite configuration
+ignite.host=192.168.131.131
```

Comment:
The Jeskin build is failing. Could you check if the build is broken or caused by something else? Thanks.